lib/db/schema/patch-072-073.sql

```diff
@@ -0,0 +1,53 @@
+SET NAMES utf8mb4 COLLATE utf8mb4_bin;
+
+-- Updates sessionWithDevice procedure to return tokenVerificationCode
+
+CREATE PROCEDURE `sessionWithDevice_12` (
```

Comment:
Updated this procedure to return the `tokenVerificationCodeHash` since the other session procedures returned this value.

Reply:
Interesting. Do we need to return it?

Presumably if nothing was left that called the other stored procedures, then there's nothing that uses `tokenVerificationCodeHash` otherwise it would be broken. Or is there something broken by the fact that it's not being returned?

No big deal either way, just figure there's no need to return data that isn't being used is all.